bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/upgrade.yaml

```diff
@@ -22,6 +21,14 @@
     airflow_releases: "{{ helm_releases.stdout | from_json | selectattr('chart', 'match', 'airflow-.*') | list | map(attribute='name') | list }}"
     database_name: "{{ (helm_releases.stdout | from_json | selectattr('chart', 'match', 'astronomer-.*') | list)[0].name | regex_replace( '-', '_') }}"
 
+- name: "Set commander upgrade timeout"
+  set_fact:
+    COMMANDER_UPGRADE_TIMEOUT: "{{ lookup('env', 'COMMANDER_UPGRADE_TIMEOUT') | default('300', True) }}"
+
+- name: "Disable deployment upgrades"
+  set_fact:
+    disable_upgrade_timeout: "{{ lookup('env', 'DISABLE_UPGRADE_DEPLOYMENT') | default('false', True) }}"
```

Comment:
@rbankston @danielhoherd shouldn't this be called disable_upgrade_deployment instead of disable_upgrade_timeout?

Reply:
fixed the variable name to disable_deployment_upgrades and also on lines 310 and 321